docs/sources/features/explore/index.md

```diff
@@ -95,6 +95,13 @@ In **Query history** tab it is also possible to filter queries by date using the
 
 > Note: If you are in split mode, filters are applied only to your currently active panel.
 
+### Search in query history
+
+You can search in your history across queries and your comments. Search is possible for queries in the Query history tab and Starred tab.
+
+1. Click the **Search queries** field.
+2. Type the term you are searching for into search field.
```

Comment:
```suggestion
1. Type the term you are searching for into search field.
```

Reply:
Per the [Markdown style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-markdown-guide.md), we want to use lazy numbering by default.